templates/default/apache_ckan.conf.erb

```diff
@@ -12,7 +12,7 @@
     WSGIPassAuthorization On
 
     # Deploy as a daemon (avoids conflicts between CKAN instances).
-    WSGIDaemonProcess ckan_default display-name=ckan_default processes=2 threads=15
+    WSGIDaemonProcess ckan_default display-name=ckan_default processes=1 threads=15
```

Comment:
is this required if we are going to double the ec2 instance size?

Reply:
Probably not essential in PROD, but it will make lower environments happier and is sensible anyway when we're running a HA setup; we already have multiple processes.

Reply:
should be a variable from the opsworks config from cfn then, same for threads 
so that we have performance tweaking without needing to update this cookbook.

Reply:
Done.